extensions/amp-analytics/amp-analytics.md

```diff
@@ -377,7 +378,7 @@ Use this configuration to fire a request when a specified element is clicked. Us
       }
     }
     ```
-
+In addition to the variables provided as part of triggers you can also specific additional / overrides for [variables as data attribute](./analytics-vars.md#variables-as-data-attribute). If used, these data attribute has to be part of element specified as the `selector`
```

Comment:
In addition to the variables provided as part of triggers, you can also override or specify additional [variables as data attributes](./analytics-vars.md#variables-as-data-attribute). If used, these data attributes have to be part of element specified as the `selector`. Same above.

@rudygalfi Can you take a look at the docs changes?